packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -466,8 +457,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   }
 
   // Caret Painters:
-  // The floating painter. This painter paints the regular caret as well.
-  late final _FloatingCursorPainter _caretPainter = _FloatingCursorPainter(_onCaretChanged);
+  // A single painter for both the regular caret and the floating cursor.
```

Comment:
Should "caret" be "cursor" here as well for consistency?

Reply:
I tried to keep the (sort of) consistency of calling "caret" the regular caret displayed all the time and keeping "cursor" for the floating cursor only (say it's an own name...)